src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -415,3 +403,42 @@ def _fetch_studio_specs(self, model_specs: JumpStartModelSpecs) -> Dict[str, Any
             Bucket=utils.get_jumpstart_content_bucket(self.region), Key=key
         )
         return json.loads(response["Body"].read().decode("utf-8"))
+
+
+    def scan_and_tag_models(self) -> None:
```

Comment:
Update to `tag_unsupported_hub_content` please

Reply:
Same comment as above